packages/@react-spectrum/searchwithin/src/SearchWithin.tsx

```diff
@@ -32,19 +32,20 @@ function SearchWithin(props: SpectrumSearchWithinProps, ref: FocusableRef<HTMLEl
     'aria-labelledby': ariaLabelledby
   } = props;
 
-  let inputRef = useRef<HTMLInputElement | HTMLTextAreaElement>();
-  let domRef = useFocusableRef(ref, inputRef);
+  let domRef = useFocusableRef(ref);
 
   // Measure the width of the field to inform the width of the menu.
   let [menuWidth, setMenuWidth] = useState(null);
   let {scale} = useProvider();
 
   let onResize = useCallback(() => {
     if (domRef.current) {
-      let width = domRef.current.offsetWidth;
-      setMenuWidth(width);
+      let width = domRef.current.querySelector('[role=group]')?.offsetWidth;
```

Comment:
I think we need to have a fallback of `domRef.current.offsetWidth` for when a label isn't passed to the SearchWithin since querySelector only matches descendants right?

Reply:
Could also make a ref on the group element for a direct reference